tests-functional/tests/test_delete_keypair.py

```diff
@@ -7,29 +7,29 @@
 @pytest.mark.rpc
 class TestDeleteKeypair:
 
-    @pytest.fixture(autouse=True)
-    def setup_backends(self, backend_new_profile):
-        self.account = backend_new_profile("sender")
+    @pytest.fixture()
+    def account(self, backend_new_profile):
```

Comment:
I see that this naming was here before, but `account` is quite misleading, I'd rather make it generic `backend` in such cases:
```suggestion
    def backend(self, backend_new_profile):
```

Reply:
renamed here and in all other places